.generator/src/generator/templates/util.j2

```diff
@@ -42,3 +42,19 @@ export type AttributeTypeMap = {
     format?: string;
   };
 }
+
+export async function compressZstd(body: string): Promise<ArrayBuffer> {
+  return await new Promise((resolve) => {
+    let zstdLib;
+    try {
+      zstdLib = require("zstd-codec")
+    } catch (error) {
+      throw new Error("zstd-codec package is missing")
+    }
+
+    zstdLib.ZstdCodec.run((zstd: any) => {
+      const simple = new zstd.Simple();
```

Comment:
I think we should cache that object, otherwise it will create it on every request and it somewhat defeats the purpose.